client/pages/airing.jsx

```diff
@@ -19,7 +19,7 @@ export default class Airing extends React.Component {
               return (
                 <div key={obj.mal_id} className='col-sm airing-anime'>
                   <img className='airing-anime-img' src={`${obj.images.jpg.image_url}`}></img>
-                  <h4 className='airing-anime-title'>{obj.title}</h4>
+                  <a href={'#details?anime=' + obj.title} className='airing-anime-title' onClick={this.props.getAnimeClicked}>{obj.title}</a>
```

Comment:
Instead of setting state when the user clicks an anime, change hashes to the anime's id

```jsx
<a href={'#details?animeId=' + obj.mal_id} className='airing-anime-title'>{obj.title}</a>
```

Then, when App re-renders due to the hash change, get the animeId from the route params

```js
if (route.path === 'details') {
  return <Details animeId={route.params.get('animeId')} />
}
```

And have Details fetch the anime info in its `componentDidMount`